src/test/java/org/broadinstitute/hellbender/tools/HtsgetReaderIntegrationTest.java

```diff
@@ -0,0 +1,129 @@
+package org.broadinstitute.hellbender.tools;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.testutils.ArgumentsBuilder;
+import org.broadinstitute.hellbender.testutils.IntegrationTestSpec;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Map;
+
+import com.google.common.collect.ImmutableMap;
+
+public class HtsgetReaderIntegrationTest extends CommandLineProgramTest {
+
+    private static final String ENDPOINT = "https://htsget.ga4gh.org/reads";
+
+    // Test ability to initiate download with no aditional parameters
+    @Test
+    public void testBasicDownload() throws IOException {
```

Comment:
We need a test using the parallel code even if it's just a tiny file like this.

Reply:
Added.